daemon/src/engine/client/cg_api.h

```diff
@@ -221,6 +221,8 @@ void            trap_SetMouseMode( MouseMode mode );
 void            trap_S_StopBackgroundTrack();
 void            trap_R_RemapShader( const char *oldShader, const char *newShader, const char *timeOffset );
 bool        trap_GetEntityToken( char *buffer, int bufferSize );
+void 			trap_SendMessage(uint8_t *buf, size_t buflen);
```

Comment:
No reason vector.data() can't be used by the gamecode if it wants to. Keep it general?